src/sinks/splunk_hec.rs

```diff
@@ -123,6 +123,7 @@ impl HttpSink for HecSinkConfig {
         } else {
             chrono::Utc::now().timestamp()
         };
+        let sourcetype = event.get(&event::log_schema().source_type_key()).cloned();
```

Comment:
Why is `sourcetype` extracted but not removed?  This seems inconsistent with the behaviour of `timestamp` directly above.

Reply:
For only a minor reason, so the field is available for indexing through `indexed_fields` option. The same is true for `host` field above `timestamp`. And i assume the `timestamp` is instead removed because it isn't useful for indexing.